css/scss/_page-home.scss

```diff
@@ -0,0 +1,72 @@
+.sfh-wrapper {
+  margin: auto 0;
+  padding: 50px 50px;
+}
+
+.sfh-title {
+  @include primaryFontStack;
+  font-size: 14px;
+  text-transform: uppercase;
+  color: $white;
+  letter-spacing: 0.1em;
+}
+
+.sfh-name {
+  @include primaryFontStack;
+  font-size: 36px;
+  color: $white;
+}
+
+.sfh-copy {
+  @include primaryFontStackLight;
+  font-size: 18px;
+  color:  $white;
+}
+
+.exercise-buttons {
+  padding: 25px 50px;
+  text-align: center;
+
+  @supports (grid-area: auto) {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-gap: 10px;
+  }
+
+  @include respond-screen ($bp-md) {
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 30px;
+  }
+}
+
+.button-one {
+  background: $button-yellow; 
+}
+.button-two {
+  background: $button-red;
+}
+.button-three {
+  background: $button-teal;
+}
+
+.button-one,
+.button-two,
+.button-three {
+  color: $black8;
+  display: inline-block;
+  border-radius: 0;
+  padding: 10px 18px;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: 700;
+  letter-spacing: 1px;
+  -moz-transition: all 0.5s;
+  -webkit-transition: all 0.5s;
+  transition: all 0.5s;
```

Comment:
@praxoc, it would be more efficient to transition only the property(ies) that you want affected. Instead of transition: **all** you can use transition: **background** when you just change the "background" property on hover.